transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/ErrorCode.kt

```diff
@@ -83,6 +83,14 @@ sealed class ErrorCode(val code: Int) {
 
     data object DeploymentIdMismatch : ErrorCode(6027) // ErrorCode needed to support proper QA automation. Not expected to happen in production.
 
+    sealed class RateLimitError : ErrorCode(6028) {
```

Comment:
Rate limit is not in scope for this task. Should be done under: https://inindca.atlassian.net/browse/MTSDK-714